### DISCUSSION TITLE
[vcpkg] Print used time on port build failure

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#17817] [Ideas]
Just as the title said. Please print the time on port build failure so I know how long I have to wait until I hit the same error again ;)

**Comment by Neumann-A** [comment]
And yeah the debug timer has an overflow condition:
`[DEBUG] cmd_execute_and_stream_data() returned 1 after -2147483648 us`

**Comment by linquize** [comment]
https://github.com/microsoft/vcpkg/blob/d4a415b7fa3f0d0ea14a789d5778abfc666ef2d0/toolsrc/src/vcpkg/base/system.process.cpp#L620
cast the duration to to `int` so overflow 32-bit integer.

**Comment by strega-nil** [comment]
Would one of you like to solve this issue?
